metomi/rose/popen.py

```diff
@@ -259,7 +258,7 @@ def run_ok(self, *args, **kwargs):
         ret_code, stdout, stderr = self.run(*args, **kwargs)
         if ret_code:
             if stderr:
-                stderr = stderr.decode()
+                stderr = stderr
             else:
                 stderr = ''
```

Comment:
These lines are redundant now, I believe

Reply:
I think that you are right.